packages/turbo-codemod/src/transforms/add-default-outputs.ts

```diff
@@ -0,0 +1,53 @@
+import { Flags } from "../types";
+import path from "path";
+import fs from "fs-extra";
+import { error, ok, skip } from "../logger";
+
+const DEFAULT_OUTPUTS = ["dist/**", "build/**"];
+
+interface TaskDefinition {
+  outputs: [];
+}
+
+interface PipelineConfig {
+  [taskName: string]: TaskDefinition;
+}
+
+export default function addDefaultOutputs(files: string[], flags: Flags) {
```

Comment:
Naming thoughts:
- specify-default-outputs?

Reply:
I followed `add-package-manager`, but maybe it should be more like like "fill-in-previously-implicit-defaults" 😆 . Ok with anything though if you have a mild or strong opinion; i have none!

Reply:
done

Reply:
I only mention codemod naming because it's also dropping `[]`. If you have a better idea go for it, otherwise this is non-blocking.

Reply:
Good point. Renamed to `set-default-outputs`